vanessaspear/passport-client
# Client can now add, view, and delete trip notes

## Description

When the user is on the trip details page, a notes text box will be present.  When they type a note and click **add** note, their note will appear in the trip notes list.  When the user clicks on the **delete** button, the note will be deleted from the list.  

## Changes
- Created a trip notes manager 
- Created a trip notes component to display the list of trip notes on the trip's details page
- Created an add note and delete note button

## Steps to test

```
git fetch origin vs-trip-notes
git checkout vs-trip-notes
```
- Login and navigate to a trip's details page 
```username=dgarcia@travel.com password: globetrotter```
- Write out a note, and click the add button
- Verify the note appears in the trip notes list
- Click the delete button, and verify the note is deleted from the list

## Related Issues
- Closes #23
